style.css

```diff
@@ -630,3 +647,430 @@
+    padding-top: -1rem;
+    margin-top: -0.5rem;
+  }
+
+  .contact-form form ul {
+    display: flex;
+    flex-direction: column;
+    margin: 1rem 10vw 1rem 10vw;
+  }
+
+  #first-sec ul {
+    display: flex;
+    margin-left: -3.5rem;
+    padding-top: -1rem;
+    margin-top: -1rem;
+  }
+
+  .div-info .sec-list li {
+    float: left;
+    margin: 1vw;
+    background-color: #ebebff;
+    padding: 0.5rem 1rem 0.5rem 1rem;
+    border-radius: 8px;
+    height: 2vw;
+  }
+
+  .flex .flex-box .social-img li {
+    float: left;
+    margin: 1vw;
+  }
+
+  .flex-3 .grid .grid-2 .language li {
+    float: left;
+    background-color: #f7f7f9;
+    margin: 1vw;
+    font-size: 15px;
+    color: #253858;
+    width: 9vw;
+    height: 9vw;
+    border-radius: 4px;
+  }
+
+  #first-sec ul li {
+    margin: 0.4rem 1rem 0 1rem;
+  }
+
+  .work-sec {
+    display: none;
+  }
+
+  .desktop-work-sec {
+    display: grid;
+    gap: 3.5rem;
+    grid-template-columns: 1;
+    grid-template-rows: repeat(4, 40rem);
+    height: 90rem;
+    margin: 5rem 6vw 50vh 6vw;
+  }
+
+  .desk-first-div,
+  .desk-second-div,
+  .desk-third-div,
+  .desk-fourth-div,
+  .desktop-work-sec,
+  .desk-div-img img {
+    grid-column: 1;
+    width: 40vw;
+    height: 30vw;
+  }
+
+  .desktop-work-sec .desk-first-div {
+    grid-column: 1;
+    grid-row: 1;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    gap: 1rem;
+    padding: 2vw;
+    width: 83vw;
+    height: 24rem;
+    border-radius: 16px;
+  }
+
+  .desktop-work-sec .desk-second-div {
+    grid-column: 1;
+    grid-row: 2;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    gap: 1rem;
+    padding: 2vw;
+    width: 83vw;
+    height: 24rem;
+    border-radius: 16px;
+  }
+
+  .desktop-work-sec .desk-third-div {
+    grid-column: 1;
+    grid-row: 3;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    gap: 1rem;
+    padding: 20px;
+    width: 83vw;
+    height: 24rem;
+    border-radius: 16px;
+  }
+
+  .desktop-work-sec .desk-fourth-div {
+    grid-column: 1;
+    grid-row: 4;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    gap: 1rem;
+    padding: 20px;
+    width: 83vw;
+    height: 24rem;
+    border-radius: 16px;
+  }
+
+  .desktop-work-sec .div-info {
+    height: 30vw;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .desktop-work-sec .div-info h2 {
+    font-size: var(--font-size-h2-desktop);
+    font-family: var(--font-family);
+    color: #091e42;
+    padding: 5px;
+    margin-top: -0.5vw;
+    margin-left: 0.8rem;
+  }
+
+  .desktop-work-sec .div-info .ul-li {
+    display: flex;
+    justify-content: space-around;
+    font-family: var(--font-family);
+    font-size: 18px;
+    color: #6b778c;
+    margin-right: 9vw;
+    margin-left: -2.6vw;
+    position: relative;
+    margin-top: -2vw;
+  }
```

Comment:
- [ ] Please ☺️ reduce the padding on the tech stack of the cards to make it look like [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Z0umLtHjW2jvzRBK-0)  

**_Figma_**

<img width="211" alt="Screenshot 2023-02-23 at 10 10 33 PM" src="https://user-images.githubusercontent.com/44798044/220979796-e2f4c639-b369-4f8e-9b14-dc83d0aae5b9.png">

**_Yours_**

<img width="413" alt="Screenshot 2023-02-23 at 9 34 22 PM" src="https://user-images.githubusercontent.com/44798044/220979959-5760e04b-bb0a-4e92-bac0-42a243b6ba37.png">